website/content/api-docs/system/inspect/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+layout: api
+page_title: /sys/internal/inspect - HTTP API
+description: >-
+  This endpoint is intended to inspect a specific internal subsystem for debugging purposes. It can be accessed with a root token or sudo privileges
+  This endpoint is off by default. See the
+  [Vault configuration documentation](/docs/configuration) to
+  enable.
+---
+
+# `/sys/internal/inspect`
+
+The `/sys/internal/inspect` is intended to inspect a specific internal subsystem for debugging purposes.
+It can be accessed with a root token or sudo privileges.
+
+
+## Supported Inspection Paths
+
+- [Router](/api-docs/system/internal/inspect/router)
```

Comment:
In the vercel preview, this link is broken - I think because the `path` in the api-docs-nav json doesn't have the `internal` part of the link.

Reply:
Fixed this in the current commit